pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/PillarboxCastPlayer.kt

```diff
@@ -162,14 +165,20 @@ class PillarboxCastPlayer internal constructor(
     override fun getState(): State {
         val remoteMediaClient = remoteMediaClient ?: return State.Builder().build()
         val mediaStatus = remoteMediaClient.mediaStatus
+        val contentPositionMs = remoteMediaClient.getContentPositionMs()
+        val contentDurationMs = remoteMediaClient.getContentDurationMs()
         val isCommandSupported = { command: Long -> mediaStatus?.isMediaCommandSupported(command) == true }
         val currentItemIndex = remoteMediaClient.getCurrentMediaItemIndex()
         val isPlayingAd = mediaStatus?.isPlayingAd == true
         val itemCount = remoteMediaClient.mediaQueue.itemCount
         val hasNextItem = !isPlayingAd && currentItemIndex + 1 < itemCount
         val hasPreviousItem = !isPlayingAd && currentItemIndex - 1 >= 0
-        val hasNext = hasNextItem // TODO handle like describe in Player.seekToNext
-        val hasPrevious = hasPreviousItem // TODO handle like describe in Player.seekToPrevious
+        val canSeek = itemCount > 0 && !isPlayingAd && isCommandSupported(MediaStatus.COMMAND_SEEK) && contentDurationMs != C.TIME_UNSET
+        val canSeekBack = canSeek && contentPositionMs != C.TIME_UNSET && contentPositionMs - seekBackIncrementMs > 0
+        val canSeekForward = canSeek && contentPositionMs + seekForwardIncrementMs < contentDurationMs
+        val hasNext = hasNextItem || canSeek
+        val hasPrevious = hasPreviousItem || canSeek
+
         val availableCommands = PERMANENT_AVAILABLE_COMMANDS.buildUpon()
```

Comment:
Maybe we should extract this into a dedicated method (`buildAvailableCommands()`)?

Reply:
Yes we could, we can handle that in the main story